chat-plugins/info.js

```diff
@@ -2091,6 +2091,10 @@ const commands = {
 				// Translated analysis do not support automatic redirects from a speciesid to the proper page
 				let pageid = pokemon.name.toLowerCase().replace(' ', '_');
 				this.sendReplyBox(`<a href="https://www.smogon.com/translations/${supportedLanguages[room.language]}/analyses/ou/${pageid}">${generation.toUpperCase()} ${Chat.escapeHTML(formatName)} ${pokemon.name} analysis</a>, brought to you by <a href="https://www.smogon.com">Smogon University</a>`);
+			} else if (formatId === 'ou' && generation === 'sm') {
+				let pageid = pokemon.name.toLowerCase().replace(' ', '_');
+				this.sendReplyBox(`<a href="https://www.smogon.com/dex/${generation}/pokemon/${speciesid}${(formatId ? '/' + formatId : '')}">${generation.toUpperCase()} ${Chat.escapeHTML(formatName)} ${pokemon.name} analysis</a>, brought to you by <a href="https://www.smogon.com">Smogon University</a>`);
+				this.sendReplyBox(`Other languages: <a href="https://www.smogon.com/translations/es/analyses/ou/${pageid}">es</a>, <a href="https://www.smogon.com/translations/fr/analyses/ou/${pageid}">fr</a>, <a href="https://www.smogon.com/translations/ita/analyses/ou/${pageid}">ita</a>, <a href="https://www.smogon.com/translations/ger/analyses/ou/${pageid}">ger</a>, <a href="https://www.smogon.com/translations/por/analyses/ou/${pageid}">por</a>`);
```

Comment:
I would probably write out the language names; it's not like you're using consistent abbreviations anyway.

```suggestion
				this.sendReplyBox(`Other languages: <a href="https://www.smogon.com/translations/es/analyses/ou/${pageid}">Español</a>, <a href="https://www.smogon.com/translations/fr/analyses/ou/${pageid}">Français</a>, <a href="https://www.smogon.com/translations/ita/analyses/ou/${pageid}">Italiano</a>, <a href="https://www.smogon.com/translations/ger/analyses/ou/${pageid}">Deutsch</a>, <a href="https://www.smogon.com/translations/por/analyses/ou/${pageid}">Português</a>`);
```

Reply:
Also wow Smogon doesn't consistently use ISO-639-1 or ISO-639-2, but rather a mix. That's annoying.